services/publicapi/test/public_api_test.go

```diff
@@ -86,6 +86,45 @@ func TestSendTransaction_BlocksUntilTransactionCompletes(t *testing.T) {
 	})
 }
 
+func TestSendTransaction_TimesOut(t *testing.T) {
```

Comment:
why did you add this test?

Reply:
There wasn't one and I needed to check if the timeout works as expected.